lib/bundler/mirror.rb

```diff
@@ -0,0 +1,183 @@
+module Bundler
+  class Settings
+    class Mirrors
+      def initialize(prober = nil)
+        @all = Mirror.new
```

Comment:
Not sure about this, all these are used for deciding which mirror should be returned depending on the validity and the configuration.

By making these public it would be expanding the public API of this object leaking internal implementation details.